# Cálculo - Tarefa 02

1- Calcule de forma numérica a integral da função $x^3$ avaliada entre os pontos $1$ e $2$. Identifique o número de retângulos que você usou.  
2- Calcule essa área de forma analítica.

In [1]:
# Cálculo integral e número de retângulos

# Define a função x^3 usando lambda
f = lambda x: x**3

# Intervalo de integração
a = 1
b = 2

# Número de retângulos
n = 1000

# Largura de cada retângulo
dx = (b - a) / n

# Soma das áreas dos retângulos
integral = sum(f(a + i * dx) * dx for i in range(n))

integral, n

(3.74650075, 1000)

Para calcular a integral de \( x^3 \) de forma analítica entre os pontos \( 1 \) e \( 2 \), precisamos resolver a integral definida:

\[ \int_{1}^{2} x^3 \, dx \]

A integral indefinida de \( x^3 \) é:

\[ \int x^3 \, dx = \frac{x^4}{4} + C \]

Agora, aplicamos os limites de integração:

\[ \left[ \frac{x^4}{4} \right]_{1}^{2} = \frac{2^4}{4} - \frac{1^4}{4} \]

Calculamos os valores:

\[ \frac{2^4}{4} = \frac{16}{4} = 4 \]
\[ \frac{1^4}{4} = \frac{1}{4} = 0.25 \]

Subtraímos os resultados:

\[ 4 - 0.25 = 3.75 \]

Portanto, a área sob a curva \( x^3 \) entre \( 1 \) e \( 2 \) é \( 3.75 \).